product_docs/docs/pgd/5/quickstart/quick_start_aws.mdx

```diff
@@ -154,7 +154,7 @@ By default, TPA configures Debian as the default OS for all nodes on AWS.
 
 Specify that the data nodes will be running [EDB Postgres Advanced Server v15](/epas/latest/) (`--edb-postgres-advanced 15`) with Oracle compatibility (`--redwood`). 
 
-You set the notional location of the nodes to `dc1` using `--location-names`. You then set `--pgd-proxy-routing` to `global` so that proxy routing can route traffic to all nodes in the cluster.
+You set the notional location of the nodes to `dc1` using `--location-names`. You then set `--pgd-proxy-routing` to `global` so that proxy routing can route traffic to all nodes within each location.
```

Comment:
I think this should state `local`, not `global`

Reply:
Fixed